Documentation/ExtensionInstallation/Index.rst

```diff
@@ -55,10 +63,73 @@ In the backend:
 8. Click on the icon with a :guilabel:`"+"` sign for your extension
    in the :guilabel:`"A/D"` column.
 
+Uninstall an Extension Without Composer
+=======================================
+
+If you installed TYPO3 via composer you should uninstall Extensions via composer.
+
+Check Dependencies
+------------------
+
+First find out, which other extensions and functions of your Typo3-Installation are dependent of the extension you
+want to deinstall. Major dependeancies you can find out by checking the Extension Repository. Look for the extension
+you want to deinstall and the others you have installed. Read for all of them the chapters 'Dependencies' and
```

Comment:
> the chapters 'Dependencies'

I would use a different word for chapter, e.g. section.  (if this refers to the section on the detail view page of extensions.typo3.org). This was not entirely clear from the text.